frontend/src/services/validateIsvService.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { getBackendURL } from '../utilities/utils';
+
+export const postValidateIsv = (
+  appName: string,
+  values: { [key: string]: string },
+): Promise<any> => {
+  const url = getBackendURL('/api/validate-isv');
+  const searchParams = new URLSearchParams();
+  if (appName) {
+    searchParams.set('appName', appName);
+    searchParams.set('values', JSON.stringify(values));
+  }
+  const options = { params: searchParams };
+  return axios
+    .get(url, options)
+    .then((response) => {
+      return response;
```

Comment:
optional: confirmation on your end, since the other API calls all use `response.data` ... though it doesn't appear you're doing anything with the response other than checking "success"